src/Commands/StartRoadRunnerCommand.php

```diff
@@ -120,6 +119,20 @@ protected function writeServerStateFile(
         ]);
     }
 
+    /**
+     * Provide road runner config file path.
+     *
+     * @return string
+     */
+    protected function configPath()
+    {
+        if (! $this->option('config-path')) {
+            touch(base_path('.rr.yaml'));
+            return base_path('.rr.yaml');
+        }
+        return $this->option('config-path');
```

Comment:
Would it make sense here to allow different styles of config paths to be passed?
```
php artisan octane:start --config-path=rr.prod.yml
php artisan octane:start --config-path=config/rr.prod.yml
php artisan octane:start --config-path=./config/rr.prod.yml
php artisan octane:start --config-path=../config/rr.prod.yml
php artisan octane:start --config-path=/etc/roadrunner/rr.prod.yml
```

Reply:
Thank you, I have added absolute path 58f1dcf